docs/api/StreamReceiveComplete.md

```diff
@@ -17,14 +17,38 @@ void
 
 # Parameters
 
-**TODO**
+- `Stream`: A handle to a valid stream
+- `BufferLength`: The number of bytes processed by the application
 
 # Remarks
 
 This is an asynchronous API but can run inline if called in a callback.
-The application, without setting `StreamMultiReceiveEnabled`, must ensure that one `StreamReceiveComplete` call corresponds to one `QUIC_STREAM_EVENT_RECEIVE` event.
-Duplicate `StreamReceiveComplete` calls after one `QUIC_STREAM_EVENT_RECEIVE` event are ignored silently even with different `BufferLength`.
-The `StreamMultiReceiveEnabled` mode doesn't follow this rule. Multiple `QUIC_STREAM_EVENT_RECEIVE` events can be indicated at once by `StreamReceiveComplete`. The application needs to keep track of accumulated `TotalBufferLength` with this mode.
+
+## Default behavior
+
+The application must ensure that for each `QUIC_STREAM_EVENT_RECEIVE` processed asynchronously (`QUIC_STATUS_PENDING`
+was returned from the callback), `StreamReceiveComplete` is called exactly once.
+
+Duplicate `StreamReceiveComplete` calls after a `QUIC_STREAM_EVENT_RECEIVE` event are ignored
+silently, even when a different `BufferLength` is provided.
+
+If `BufferLength` is smaller than the number of bytes indicated in the matching `QUIC_STREAM_EVENT_RECEIVE`, MsQuic will
```

Comment:
I personally prefer the extra details, but maybe it goes into too much detail? @nibanks thoughts?

Reply:
Which extra details do you feel are missing?